modules/rosa-hcp-sts-creating-a-cluster-ext-auth-cluster-cli.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-creating-a-cluster-external-auth-cluster-cli_{context}"]
+= Creating a {hcp-title} cluster that uses external authentication providers
+:source-highlighter: pygments
+:pygments-style: emacs
+:icons: font
+ 
+Use the ROSA CLI's `--external-auth-providers-enabled` flag to create your cluster that uses an external authentication service.
+
+.Procedure
+
+. Use one of the following commands to create your ROSA with HCP cluster with the `--external-auth-providers-enabled` flag included:
```

Comment:
Single step proc:
```suggestion
* Use one of the following commands to create your ROSA with HCP cluster with the `--external-auth-providers-enabled` flag included:
```

Reply:
You could probably slurp the info/note in this step into the intro of the topic, and then just have the two options as steps